pt-BR: update some fuzzy translations

joaovicmendes opened the pull request.

mgeisler commented:
Thanks @joaovicmendes! The PR looks great from a technical point of view :+1:

joaovicmendes commented on po/pt-BR.po:
````diff
@@ -3283,12 +3333,20 @@ msgid ""
+"    }\n"
+"    println!();\n"
+"}\n"
+"```"
````
My translation stopped here. The changes below came from `msgmerge`

mgeisler commented:
Hi @rastringer and @hugojacob, please look this over when you get a minute.

hugojacob commented on po/pt-BR.po:
```diff
@@ -1469,29 +1446,26 @@ msgid ""
-"   arquivos `Android.bp` em `src/android/`."
+"Você precisará de um [checkout do AOSP][1]. Faça um checkout do [repositório\n"
+"do curso][2] no mesmo computador e mova o diretório `src/android/` para a raiz\n"
+"do seu checkout doAOSP. Isso garantirá que o sistema de compilação do Android veja os\n"
```
```suggestion
"do seu checkout do AOSP. Isso garantirá que o sistema de compilação do Android veja os\n"
```

hugojacob commented on po/pt-BR.po:
```diff
@@ -784,9 +776,8 @@ msgid "Driver"
-#, fuzzy
 msgid "Using It"
-msgstr "Usando o Cargo"
+msgstr "Usando o driver"
```
```suggestion
msgstr "Usando o Cargo"
```

Here "Cargo" refers to the package manager.

joaovicmendes commented:
Hi, @hugojacob. I've updated the PR with the more recent changes to the repository, and I've fixed some syntax errors that were breaking the tests. Could you PTAL?
